forms-flow-web/src/components/Application/ApplicationFilter.js

```diff
@@ -144,13 +146,17 @@ const ApplicationFilter = ({
               value={searchParams.applicationStatus}
```

Comment:
we need to give data test id for select also and options we won't add again input keyword example ; submission-filter-applicationstatus-{option.value} this is enough